tensorboard/plugins/graph/tf_graph_app/tf-graph-app.html

```diff
@@ -181,10 +214,20 @@
       // Fetch a pbtxt file. The fetching will be part of the loading sequence.
       this.$.loader.datasets = [{
         // Just name the dataset based on the file location.
-        "name": this.pbtxtFileLocation,
-        "path": this.pbtxtFileLocation,
+        name: this.pbtxtFileLocation,
+        tags: [{
+          tag: null,
+          displayName: this.pbtxtFileLocation,
+          conceptualGraph: false,
+          opGraph: true,
+          profile: false,
+        }],
       }];
-      this.$.loader.set('selectedDataset', 0);
+      this.$.loader.selection = {
+        run: this.pbtxtFileLocation,
+        tag: null,
+        type: tf.graph.SelectionType.OP_GRAPH,
+      };
     } else if (this.pbtxt) {
```

Comment:
I think this branch also needs fixes.  In particular the private method below was renamed in #1853 from `_parseAndConstructHierarchicalGraph()` to `_fetchAndConstructHierarchicalGraph()`.  Not sure if anything else needs to change?

Reply:
Didn't think a private method would be used like this. Thanks for catching this and fixed!